### ISSUE TITLE
something wrong with mounts

### ISSUE THREAD

**Issue by Seokotov** [Sphereserver/Source-X#350]
it's happen with vanilla X and X-pack

`19:07:ERROR:Mount (UID=014a4, id=0dc 'Llama'): Can't auto-fix invalid mount item (ACTARG1=04fffffff)'
19:07:ERROR: Reason: MEMORY_IPET missing.
19:07:ERROR:Mount (UID=014a7, id=0dc 'Llama'): Can't auto-fix invalid mount item (ACTARG1=04fffffff)'

have a same errors for id of human uid of players...
mb #348 near?

**Comment by cbnolok** [comment]
Is that a mounted npc or a free, wandering one?

Il gio 12 dic 2019, 18:18 Seokotov <notifications@github.com> ha scritto:

**Comment by Seokotov** [comment]
yes, its free

**Comment by Seokotov** [comment]
have also many errors:
`00:04:ERROR:Mount (UID=04e92, id=0190 'casdaa'): Can't auto-fix invalid mount item (ACTARG1=00)'
00:04:ERROR: Reason: MEMORY_IPET missing.
00:04:ERROR:Mount (UID=02b3d, id=0190 'Ser Pounce'): Can't auto-fix invalid mount item (ACTARG1=00)'
00:04:ERROR: Reason: MEMORY_IPET missing.`

**Comment by cbnolok** [comment]
I have some ideas, i'll try this weekend

Il gio 12 dic 2019, 22:13 Seokotov <notifications@github.com> ha scritto:

**Comment by cbnolok** [comment]
Nope, can't reproduce it here, i need the exact steps to get that error

**Comment by Seokotov** [comment]
looks like just memory_ipet on players missed
how to add it to all players?
```
15:56:ERROR:Mount (UID=01c0c, id=0190 'Batman'): Can't auto-fix invalid mount item (ACTARG1=00)'
15:56:ERROR: Reason: MEMORY_IPET missing.
```

**Comment by Seokotov** [comment]
just wiped all npc, but errors still here
what mount items it mean?
`16:10:ERROR:Mount (UID=01c0c, id=0190 'Batman'): Can't auto-fix invalid mount item (ACTARG1=00)'
16:10:ERROR: Reason: MEMORY_IPET missing.`

**Comment by FiLoUs64** [comment]
I have these errors still again but with previous build no.

Problem seems to be with shrink mount and a lot of other items

> 10:04:ERROR:GC: Deleted UID=046a11, Defname='c_drazos', Name='Drazos'. Invalid code=0x1204 (Ridden NPC without a valid mount item).
10:04:ERROR:Mount (UID=046a11, id=061dc 'Drazos'): Can't auto-fix invalid mount item (ACTARG1=04016569e)'
10:04:ERROR: Reason: Mount item in owner's LAYER_HORSE has a MORE2 which doesn't match this mount UID (owner's riding another mount?).

> 10:04:ERROR:GC: Deleted UID=04006347c, Defname='i_tambourine', Name='Tambourine'. Invalid code=0x1104 (Ridden NPC without a valid mount item).

**Comment by FiLoUs64** [comment]
Error occurs after build 1.1.0.591

With 1.1.0.591 everything seems good
